recipes/ffnvcodec-headers/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = 'ffnvcodec-headers' %}
+{% set version = '8.2.15.4' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 'https://github.com/FFmpeg/nv-codec-headers/releases/download/n{{ version }}/nv-codec-headers-{{ version }}.tar.gz'
+  sha256: 9a567b45ece1930e65803411f12736d8925f0d1adbc31e962cb5c441969940f0
+
+build:
+  number: 1
+  skip: True  # [osx]
+  script: make PREFIX={{ PREFIX }} install
+
+requirements:
+  build:
+    # git provides the sed command
+    - git   # [win]
```

Comment:
I wonder if we can use one of the msys smaller packages to get `sed`...

Reply:
That’s what somebody had suggested. I tried one of them, but it complained that /tmp wasn’t available. I can iterate on that separately though. Thanks!

Reply:
That is not important. I was just wondering if that is possible. (But if you find out please let me know so I can use it :smile:)